shell/src/builtins.rs

```diff
@@ -332,6 +332,7 @@ where
 {
     let mut args = args.map(|v| v.resolve_arg(jobs).unwrap_or_default());
     let command_str = command.to_string_lossy();
+    // TODO sls how can we document builtins?
```

Comment:
Good question.  I don't have a good answer yet.  We could add a symbol to attach docs to but that would get messy.  Maybe we need a shell builtin "doc" that covers shell builtins?